### ISSUE TITLE
[Feature]: quivr with local llm

### ISSUE THREAD

**Issue by robertsmaoui** [StanGirard/quivr#1611]
### The Feature

In my opinion, it will be more interesting if quivr can be used with offline models without the need for OpenAI api key,

Is this possible with the new versions?

### Motivation, pitch

this feature saves money and allows chat confidentiality !

**Comment by StanGirard** [comment]
https://www.quivr.app/blog/is-mistral-a-good-replacement-for-openai ;)

**Comment by robertsmaoui** [comment]
1. In the demo https://www.quivr.app/, it does not work when you choose huggingface/mistralai/Mistral-7B-Instruct-v0.1 (no response in the chat)

2. In the source code, I did not find any file that mentions the use case of huggingface or environment variables for the endpoint
3. As long as we depend on the huggingface API it is not considered offline

Thank you very much StanGirard for Quivr

**Comment by cccc11231** [comment]
Hi @robertsmaoui 

Did you try this: 
https://brain.quivr.app/docs/Developers/selfHosted/run_fully_local

I am also looking for completely offline version. Do you know does the above instruction work?

I try to deploy quiver locally a few month ago, at that time, everything works except embedding part which still rely on openAI. I don't know current status. Do you know if we can deploy it completely offline or not?

Thanks.

**Comment by StanGirard** [comment]
@cccc11231 This is now working :)

**Comment by Japhys** [comment]
> @cccc11231 This is now working :)

I know this was closed but I found this when looking for some answers. the links to the docs do not work and I can't find any documentation on how to use hugging face local model. I am considering trying quivr but I can't seem to find the answers to my need. I want to chose a local multilingual transformer model from hugging face for my embeddings but want (for now) use openai api for the generation part. Is this at all possible in an easy to configure way with quivr?
